tests/test_api.py

```diff
@@ -37,14 +37,26 @@ def compare_response(response_fixture, reponse_data):
             assert actual == expected
 
 
-create_params = [
-    (mf.path, create_opts) for mf in ALL_MODEL_FIXTURES for create_opts in mf.create_opts
-]
+def create_with_dependencies(create_opts, mf, client):
+
+    if hasattr(mf, "dependencies"):
```

Comment:
My comment above would make this `if` unnecessary.

Reply:
I think I got all of these out between https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/31/commits/ace3c1a02743c1e740d01906a4bd48a32099d87a and https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/31/commits/930857c8c4f16d6d14cf42afffce02fdde11b052. Much nicer this way!